test/functional/p2p_invalid_messages.py

```diff
@@ -81,7 +81,7 @@ def run_test(self):
 
             # Peer 1, despite serving up a bunch of nonsense, should still be connected.
             self.log.info("Waiting for node to drop junk messages.")
-            node.p2p.sync_with_ping(timeout=30)
+            node.p2p.sync_with_ping(timeout=320)
```

Comment:
In upstream changes, the original timeout was 120 and changed to 320,  Is 320 ok here or shall we choose a lower